app/ui/src/app/connections/list/list.component.html

```diff
@@ -108,24 +108,17 @@ <h1>Register Applications</h1>
         </div>
       </ng-container>
       <ng-container *ngIf="showNewConnection && connections.length > 0">
-        <div class="connection new-connection col-xs-12 col-sm-3 col-md-2 card"
-             (click)="onSelect(undefined, $event)">
-          <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
-            <div class="card-pf-body">
-
-              <!-- Card Icon -->
-              <div class="card-pf-top-element">
-                <span class="card-pf-icon-large pficon pficon-add-circle-o">
-                </span>
-              </div>
-
-              <!-- Card Description / Item Overview -->
-              <div class="card-pf-info text-center">
-                <button class="btn btn-default" (click)="onSelect(undefined, $event)">Create Connection</button>
-              </div>
-            </div>
-          </div>
-        </div>
+        <syndesis-empty-state-card
+          [body]="cardBody"
+          [colClass]="'col-xs-12 col-sm-3 col-md-2'"
+          [topElement]="cardTopElement">
+          <ng-template #cardBody>
+            <button class="btn btn-default" (click)="onSelect(undefined, $event)">Create Connection</button>
+          </ng-template>
+          <ng-template #cardTopElement>
+              <span class="card-pf-icon-large pficon pficon-add-circle-o"></span>
```

Comment:
I'd maybe have this as a default template so it's not required to pass in unless customization is needed.  I think across the app the `+` with the circle around it is the standard, so maybe it's worth just simplifying your component and just move this to your component's template?